pangeo-binder/values.yaml

```diff
@@ -204,13 +204,22 @@ dask-gateway:
               operator: "Equal"
               value: "worker"
               effect: "NoSchedule"
+            - key: "k8s.dask.org/dedicated"
```

Comment:
This was wrong. Fixed in https://github.com/pangeo-data/pangeo-binder/pull/134 to be `_` rather than `/`